app/javascript/packs/analyticsDashboard.js

```diff
@@ -1,11 +1,26 @@
 import { initCharts } from '../analytics/dashboard';
 
+function renderOrgData() {
+  const organizationsArray = Array.from(
+    document.getElementsByClassName('organization'),
+  );
+  const activeOrg = organizationsArray.filter(
+    (org) => org.getAttribute('aria-current') === 'page',
+  )[0];
```

Comment:
```suggestion
  const activeOrg = organizationsArray.find(
    (org) => org.getAttribute('aria-current') === 'page',
  );
```

suggestion (not-blocking) - `find` will be a little more efficient than `filter`, as it will automatically stop when it finds the first match 🙂